client/src/components/AuditAdmin/Progress/Progress.tsx

```diff
@@ -172,20 +178,18 @@ const Progress: React.FC<IProgressProps> = ({
             )
           case JurisdictionProgressStatus.UPLOADS_IN_PROGRESS:
             return <Status intent="warning">{filesUploadedText}</Status>
-          case JurisdictionProgressStatus.UPLOADS_NOT_STARTED:
-            return (
-              <Status>
-                {auditType === 'BALLOT_POLLING'
-                  ? 'No manifest uploaded'
-                  : filesUploadedText}
-              </Status>
-            )
+          case JurisdictionProgressStatus.UPLOADS_NOT_STARTED_LOGGED_IN:
+            return <Status intent="warning">Logged in</Status>
```

Comment:
I wonder if we should make this a different color to differentiate from in-progress status (specifically thinking about the map, where color is more important). I don't think it's a blocking concern though, so happy to ship this and then reconsider it later